models/collection.js

```diff
@@ -5,8 +5,13 @@ const mutate = require('../lib/mutate');
 
 const BUILD_MODEL = require('./build').get;
 const PIPELINE_MODEL = require('./pipeline').get;
+const PRS = {
```

Comment:
We just want the numbers? Or we want the last build of opened PRs?

Reply:
According to the [mock](https://user-images.githubusercontent.com/78533/27356398-2694102e-55c3-11e7-877f-ec96d074bbf0.png) we just want the numbers

Reply:
Ah interesting. So how would it look if we had multiple PRs open?

Reply:
@r3rastogi basically it will be dependent on the number of PR jobs. If there are multiple PRs open then there will be multiple PR jobs and the number of jobs would be represented by `prs.open` in the response. For each of the PR jobs, it will then look at the last build and the number of PRs whose last builds are failing would be represented by `prs.failing`.